### DISCUSSION TITLE
Resizeable main window

### DISCUSSION THREAD

**Discussion by boardnick0815** [snappysnappydog/Simple-IP-Config#171] [Ideas]
Hi alltogether, Hi Kurtis,

first of all: Thank you for this great little tool!

Imho it would be nice if the main window could be resizeable.
If you have "long" profile names they are trimmed like this:

![resizeable](https://user-images.githubusercontent.com/28288422/193454643-2677ad3f-fb94-4c7d-a407-7f2f41d57149.png)

Im my use case my profile names sometimes have some hints in their names and if the main window / the gui part containing the profiles could be resizeable i would be able to read them directly / unshortened.

Another great thing i could make use of for my case is already mentioned by [tverhoef](https://github.com/tverhoef) in [discussion 118](https://github.com/KurtisLiggett/Simple-IP-Config/discussions/118).
But regardless of the very good suggestion to add a notes field it would be nice to have the ability to resize the main window.

Best regards
Aaron

**Comment by KurtisLiggett** [comment]
@boardnick0815 
I have fiddled with that before but never put much effort into it. After this next revision, I will try to get this one implemented.
Also, it would probably be helpful to add a tooltip, so when you hover over the names, it will pop up showing the full name.

**Comment by KurtisLiggett** [comment]
Update: resize the main has been implemented in the master branch, available in the next release v2.9.8.
